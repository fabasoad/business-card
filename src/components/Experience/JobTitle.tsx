import * as React from 'react'

interface JobTitleProps {
  company: string
  title: string
}

export default function JobTitle(props: JobTitleProps) {
  return (
    <div className="timeline-heading">
      <h4>{props.company}</h4>
      <h4 className="subheading">{props.title}</h4>
    </div>
  )
}
