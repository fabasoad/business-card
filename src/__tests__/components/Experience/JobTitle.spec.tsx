/// <reference types="jest" />
import * as React from 'react'
import { shallow } from 'enzyme'
import JobTitle from '../../../components/Experience/JobTitle'

test('should render JobTitle correctly', () => {
  const wrapper = shallow(<JobTitle company="test-company" title="test-title" />)
  expect(wrapper).toMatchSnapshot()
})
