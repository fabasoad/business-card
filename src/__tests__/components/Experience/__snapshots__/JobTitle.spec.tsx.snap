// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render JobTitle correctly 1`] = `
<div
  className="timeline-heading"
>
  <h4>
    test-company
  </h4>
  <h4
    className="subheading"
  >
    test-title
  </h4>
</div>
`;
